src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -851,6 +851,7 @@ function build_eal_list() {
 	$section->add($group);
 	$counter += 1;
 }
+$section->setHelp('Note: Blowfish, 3DES, CAST128, MD5, SHA1, and DH groups 1, 2, 22, 23, and 24 provide weak security and should not be used.');
```

Comment:
I prefer "should be avoided" rather than "should not be used" here. The latter makes it sound like we're lazy and haven't removed them when we know better, rather than the option still being there because some people need them because they are burdened with third-party compatibility or other vendor/business needs.